src/bootstrap/setup.rs

```diff
@@ -107,6 +108,19 @@ pub fn setup(src_path: &Path, profile: Profile) {
     let include_path = profile.include_path(src_path);
     println!("`x.py` will now use the configuration at {}", include_path.display());
 
+    let build = TargetSelection::from_user(&env!("BUILD_TRIPLE"));
+    let stage_path = ["build", build.rustc_target_arg(), "stage1"].join("/");
```

Comment:
You need to create this directory first, otherwise it will give an error:
```
$ rustup toolchain link stage1 build/x86_64-unknown-linux-gnu/stage1
error: not a directory: 'build/x86_64-unknown-linux-gnu/stage1/lib'
```

Reply:
Do you think we should also check if `rustup` has been installed and inform the user if it hasn't?

Reply:
Sure, doing that if the profile isn't "user" seems helpful :)

Reply:
Is there a best practice for getting this information?

Reply:
The profile is passed in as an argument to `setup`.